power-supply/power_supply.cpp

```diff
@@ -704,7 +689,9 @@ void PowerSupply::updateInventory()
 
     assetProps.emplace(SN_PROP, sn);
     assetProps.emplace(PN_PROP, pn);
+#ifdef UCD90160_DEVICE_ACCESS
     assetProps.emplace(MODEL_PROP, ccin);
+#endif
     interfaces.emplace(ASSET_IFACE, std::move(assetProps));
 
     versionProps.emplace(VERSION_PROP, version);
```

Comment:
Does MIhawk PSU have the version?

Reply:
@alvintpwang Yes, mihawk's psu has the version.
Originally I had added the information of "manufacturing date", because psu-monitor wrote the information in inventory, so I had to change to Asset.yaml of phosphor-dbus-interfaces if I wanted to add  "manufacturing date".
So I didn’t add MFRDate on this version.